libs/core/algorithms/include/hpx/algorithms/traits/is_relocatable.hpp

```diff
@@ -13,8 +13,7 @@ namespace hpx {
     template <typename T>
     struct is_relocatable
     {
-        static constexpr bool value =
-            std::is_move_constructible_v<T> && std::is_destructible_v<T>;
+        static constexpr bool value = std::is_move_constructible_v<T>;
     };
```

Comment:
Almost certainly you should make this
```
template <class T>
struct is_relocatable : std::is_move_constructible<T> {};
```
or
```
template <class T>
struct is_relocatable : std::bool_constant<std::is_move_constructible_v<T>> {};
```
so that `is_relocatable<X>` can be used for tag dispatch.

Reply:
makes sense